SUPPORTED-VERSIONS.md

```diff
@@ -5,13 +5,18 @@
 | **iOS**    |     ✅    |  `11+` |
 | **tvOS**   |     ✅    |  `11+` |
 | **iPadOS** |     ✅    |  `11+` |
-| **VisionOS** |   ⚠️    |  `1.1+` |
+| **visionOS** |   ⚠️    |  `1.0+` |
+| **macOS**  |     ⚠️    |  `10.15+` |
 | **watchOS**|     ❌    |  `n/a` |
-| **macOS**  |     ❌    |  `n/a` |
 | **Linux**  |     ❌    |  `n/a` |
 
 ## VisionOS
-VisionOS is not officially supported by Datadog SDK. Some features may not be fully functional.
+
+VisionOS is not officially supported by Datadog SDK. Some features may not be fully functional. Note that `CrashReporting` is not supported on VisionOS, due to lack of support on the [PLCrashReporter side](https://github.com/microsoft/plcrashreporter/issues/288).
+
+## MacOS
+
+MacOS is not officially supported by Datadog SDK. Some features may not be fully functional. Note that `RUM`, which heavily depends on `UIKit` does not build on macOS.
```

Comment:
If we know this is not going to build, we must not add RUM support in project. This will cause unnecessary friction.

Reply:
We can only have one Package definition and as far as I know, and we can't do selective destination (Session Replay is iOS only for example)